ci-operator/templates/openshift/openshift-azure/build-node-image.yaml

```diff
@@ -133,6 +137,13 @@ objects:
         sed -i -e "s#TENANT_ID#TENANT#" /tmp/shared/credentials
         sed -i -e "s#CLIENT_SECRET#SECRET#" /tmp/shared/credentials
         source /tmp/shared/credentials
+        
+        # after 3.10 decommission this will become default
+        if [ ${OPENSHIFT_RELEASE} == "3.11" ] ; then
+            AZURE_REPO_URL="https://rpms.svc.ci.openshift.org/openshift-origin-v3.11/"
+        else
+            AZURE_REPO_URL=$(curl -qs https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/release-${OPENSHIFT_RELEASE}/.latest-rpms)
```

Comment:
I guess you can hardcode 3.10 here, there is no reason to have this being configurable.